build/template-Build-run-tests-sign.yml

```diff
@@ -28,7 +28,7 @@ steps:
 - task: UseDotNet@2
   displayName: 'Use .Net Core SDK 9.x'
   inputs:
-    version: 9.0.100-preview.4.24267.66
+    version: 9.0.100-preview.7.24407.12
```

Comment:
```suggestion
    version: 9.0.0-rc.1.24431.7
```

Reply:
would like @kellyyangsong to take this locally and make sure the tests work locally instead of taking the suggestion via the UI.

Reply:
Whatever works best.

Reply:
Updated to rc1.